v1/lavender.json

```diff
@@ -0,0 +1,173 @@
+{
+  "name": "Xiaomi Redmi Note 7",
+  "codename": "lavender",
+  "unlock": ["confirm_model", "unlock"],
+  "user_actions": {
+    "recovery": {
+      "title": "Reboot to Recovery",
+      "description": "With the device powered off, hold Volume Down + Power.",
+      "image": "phone_power_down",
+      "button": true
+    },
+    "bootloader": {
+      "title": "Reboot to Bootloader",
+      "description": "With the device powered off, hold Volume Up + Power.",
+      "image": "phone_power_up",
+      "button": true
+    },
+    "boot": {
+      "title": "Boot the device",
+      "description": "Power on the device.",
+      "image": "phone_power_up",
+      "button": true
+    },
+    "confirm_model": {
+      "title": "Confirm your model",
+      "description": "Please double-check that your device is a Xiaomi Redmi Note 7 (lavender)."
+    },
+    "unlock": {
+      "title": "OEM unlock",
+      "description": "If you haven't done so already, make sure to OEM unlock your device first.",
+      "link": "https://en.miui.com/unlock"
```

Comment:
That doesn't seem to work. What unlock method does it use?

Reply:
fastboot, but requires Xiaomi account to be logged in first and wait for 7 days.

And the website is kind of weird, I had to add a slash at the end of it in order for it to work.. https://en.miui.com/unlock/

Reply:
I just updated the URL so that the site works.